source/scenes/external/SignatureRequest/index.tsx

```diff
@@ -38,10 +39,10 @@ const SignatureRequest = () => {
   const { data: stringData } = queryString.parse(location.search);
 
   const { signatureRequestEncoded, asset }:
-    {  signatureRequestEncoded: string, asset: string } = JSON.parse(stringData as string);
+    { signatureRequestEncoded: string, asset: string } = JSON.parse(stringData as string);
```

Comment:
This was already in the codebase but I'm just noticing now... can we put these two JSON.parse calls inside a try/catch? If the wrong format of data was sent then we'd crash the app here without the try/catch.

Reply:
Not exactly... Verification is done in requests.ts... And this JSON.parse is from the data sent by us to the screen...

Reply:
Here...
https://github.com/StardustCollective/stargazer-wallet-ext/blob/16da0111f15005db06963dcd4cc6653836961f96/source/scripts/Background/controllers/MessageHandler/requests.ts#L54-L60

Reply:
Yeah, but I mean, those details are sent by us... we would notice while coding...

Reply:
This happens as well in other screens
https://github.com/StardustCollective/stargazer-wallet-ext/blob/e3c4b198132e68e3a8a6ca18f4616ba4a4c296b8/source/scenes/external/ApproveSpend/ApproveSpend.tsx#L63-L67